cmake/FindKissFFT.cmake

```diff
@@ -47,17 +46,29 @@ if(NOT KISS_FFT_FOUND)
         
                       GIT_REPOSITORY  https://github.com/AchimTuran/KissFFT
                       GIT_TAG         master)
-            
-  set(KISS_FFT_FOUND TRUE)
-  set(KISS_FFT_LIBRARIES ${CMAKE_BINARY_DIR}/depends/output/lib/KissFFT.lib)
-  set(KISS_FFT_INCLUDE_DIRS ${CMAKE_BINARY_DIR}/depends/output/include)
+
+  set(KISS_FFT_INCLUDE_DIR ${CMAKE_BINARY_DIR}/depends/output/include)
+  set(KISS_FFT_LIBRARY ${CMAKE_BINARY_DIR}/depends/output/lib/KissFFT.lib)
+
+  # Workaround for: https://cmake.org/Bug/view.php?id=15052
+  file(MAKE_DIRECTORY ${KISS_FFT_INCLUDE_DIR})
 endif()
 
-# handle the QUIETLY and REQUIRED arguments and set KISS_FFT_FOUND to TRUE if
-# all listed variables are TRUE
-include("FindPackageHandleStandardArgs")
-find_package_handle_standard_args(KISS_FFT DEFAULT_MSG KISS_FFT_INCLUDE_DIRS KISS_FFT_LIBRARIES)
-mark_as_advanced(KISS_FFT_INCLUDE_DIRS KISS_FFT_LIBRARIES KISS_FFT_DEFINITIONS KISS_FFT_FOUND)
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(KISS_FFT DEFAULT_MSG KISS_FFT_INCLUDE_DIR KISS_FFT_LIBRARY)
+mark_as_advanced(KISS_FFT_INCLUDE_DIR KISS_FFT_LIBRARY)
 
-message(STATUS "KISS_FFT_INCLUDE_DIRS=${KISS_FFT_INCLUDE_DIRS}")
-message(STATUS "KISS_FFT_LIBRARIES=${KISS_FFT_LIBRARIES}")
+if(KISS_FFT_FOUND)
+  set(KISS_FFT_INCLUDE_DIRS ${KISS_FFT_INCLUDE_DIR})
+  set(KISS_FFT_LIBRARIES ${KISS_FFT_LIBRARY})
```

Comment:
we don't really use KISS_FFT_INCLUDE_DIRS or KISS_FFT_LIBRARIES anywhere, so we could also remove them. New approach is to just use the target we generate below which does the dependency management automatically.